libs/amp-evaluation/.env.example

```
@@ -12,8 +12,10 @@
 # ============================================================================
 # Identifies the agent being evaluated
 
-AMP_AGENT_UID=my-agent
-AMP_ENVIRONMENT_UID=production
+AMP_NAMESPACE=default
+AMP_PROJECT=my-project
+AMP_COMPONENT=my-agent
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AMP_COMPONENT key should go before the AMP_NAMESPACE key

(UnorderedKey)